### PR TITLE
LPD-41874 Unable to exact match Search in Objects using double quotes

### PR DESCRIPTION
**Pull request by abesousa**
Related issue: https://issues.liferay.com/browse/LPD-41874

### CODE REVIEW

**Comment by abesousa** [comment]
ci:test:sf

**Review by marcelabc — CHANGES_REQUESTED** [review]
Hi 
I finished my review. Let me know if you have questions :smile:

File: modules/apps/object/object-service/src/main/java/com/liferay/object/internal/search/spi/model/query/contributor/ObjectEntryKeywordQueryContributor.java [hunk 1]
```diff
@@ -111,37 +111,52 @@ public void contribute(
...
 		}
 
-		for (String token : _tokenizeKeywords(keywords)) {
-			if (addObjectEntryTitle.get() && !Validator.isBlank(token)) {
-				try {
-					booleanQuery.add(
-						new TermQueryImpl(Field.ENTRY_CLASS_PK, token),
-						BooleanClauseOccur.SHOULD);
+		String titleField = "objectEntryTitle";
+		QueryConfig queryConfig = searchContext.getQueryConfig();
```
Comment (marcelabc) [inline modules/apps/object/object-service/src/main/java/com/liferay/object/internal/search/spi/model/query/contributor/ObjectEntryKeywordQueryContributor.java:115]: sort

**Comment by abesousa** [comment]
ci:test:sf

**Comment by abesousa** [comment]
ci:test:sf

**Review by marcelabc — APPROVED** [review]
LGTM 🎉

**Comment by marcelabc** [comment]
ci:test:object

**Comment by abesousa** [comment]
ci:report:77098253

**Comment by abesousa** [comment]
@marcelabc I executed the tests with unique failures locally and they are all good.

**Comment by abesousa** [comment]
ci:forward

**Pull request closed without merging**